test/new-e2e/tests/process/linux_test.go

```diff
@@ -268,6 +268,39 @@ func (s *linuxTestSuite) TestProcessChecksInCoreAgent() {
 	requireProcessNotCollected(t, payloads, "process-agent")
 }
 
+func (s *linuxTestSuite) TestProcessChecksInCoreAgentWLMProcessCollector() {
```

Comment:
These test names are getting a bit long nowadays 😓 Maybe just `TestProcessChecksWLM` is enough